### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by adamkudrna**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "stylelint-config-visionapps-order",
-  "version": "0.3.1",
+  "name": "@visionappscz/stylelint-config-visionapps-order",
```
Comment (adamkudrna) [inline package.json]: @bedrich-schindler You did the same thing (adding namespace) with [eslint-config-visionapps](https://github.com/visionappscz/eslint-config-visionapps). Is there anything else needed?

File: package.json [hunk 2]
```diff
@@ -14,23 +14,28 @@
...
+    "eslint-plugin-jsx-a11y": "^6.1.1",
+    "eslint-plugin-react": "^7.11.1",
+    "jest": "^24.0.0",
+    "stylelint": "^9.10.1",
+    "stylelint-order": "^2.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^9.2.0",
-    "stylelint-order": "~0.8.1"
+    "stylelint": "^9.10.1",
```
Comment (adamkudrna) [inline package.json]: @crishpeen Perhaps `^9.0.0` is just fine?

Comment (adamkudrna) [reply package.json]: OK, I changed it. `stylelint-config-standard` has it the same way.

File: package.json (hunk 1 above)
Comment (bedrich-schindler) [reply package.json]: @adamkudrna This is the only thing that need to be changed in code. So it is correct. But when new version is released, please deprecate the old one like I did it with eslint-visionapps-config.

Docs: https://docs.npmjs.com/cli/deprecate

**Pull request merged**